src/state/opamFileTools.ml

```diff
@@ -1192,6 +1192,10 @@ let read_opam dir =
       (OpamPp.string_of_bad_format (OpamPp.Bad_format (snd err)));
     None
   | None, None -> None
+  | exception (OpamPp.Bad_version _bf) ->
+    log "opam-version unsupported on %s. Add as dummy unavailable package."
```

Comment:
`snd _bf` I think is the version, which is worth logging, so I think
```suggestion
  | exception (OpamPp.Bad_version (_, version) ->
    log "opam-version %S unsupported on %s. Added as dummy unavailable package." version
```
(s/Add/Added to describe what it has done, rather than what opam might be expecting the user to do)